packages/flutter/test/material/chip_theme_test.dart

```diff
@@ -0,0 +1,220 @@
+// Copyright 2018 The Chromium Authors. All rights reserved.
+// Use of this source code is governed by a BSD-style license that can be
+// found in the LICENSE file.
+
+import 'dart:ui' show window;
+
+import 'package:flutter/material.dart';
+import 'package:flutter/rendering.dart';
+import 'package:flutter_test/flutter_test.dart';
+import 'package:flutter/painting.dart';
+
+import '../rendering/mock_canvas.dart';
+
+RenderBox getMaterialBox(WidgetTester tester) {
```

Comment:
It doesn't seem like this method should be called get Material?

Reply:
No, it should, because it gets the box that is rendering the Material widget, which has the background color in it.  Added a comment.